### ISSUE TITLE
libgpu backend with CuDNN won't build (Windows)

### ISSUE THREAD

**Issue by jtc42** [Theano/Theano#5776]
Fresh install of Anaconda with Python 3.6, on Windows 10 64-bit. libpython and m2w64-toolchain have both been installed using conda install.

Upon importing Theano using the new libgpu backend, I get the following error:

```
Can not use cuDNN on context None: cannot compile with cuDNN. We got this error:

In file included from C:\\CUDA\\v8.0\\include/driver_types.h:53:0,      
from C:\\CUDA\\v8.0\\include/cudnn.h:63,
from C:\\Users\\jtc92\\AppData\\Local\\Temp\\try_flags_733_4ik6.c:4:
C:\\CUDA\\v8.0\\include/host_defines.h:84:0: warning: "__cdecl" redefined
#define __cdecl <built-in>: 
note: this is the location of the previous definition

C:/Anaconda3/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/5.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: 
cannot find -lcudnn\r\ncollect2.exe: error: ld returned 1 exit status

Mapped name None to device cuda: GeForce GTX 1060 3GB (0000:01:00.0)
```

My .theanorc is set up as:
```
[global]
floatX = float32
device = cuda

compiler_bindir=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin

[cuda]
root = C:\CUDA\v8.0

[dnn]
library_path = C:\CUDA\v8.0\lib
include_path = C:\CUDA\v8.0\include
```
CuDNN is installed to the default CUDA directory, as above.

Any advice would be great. Weirdly, using the old CUDA backend works perfectly, but this is obviously not an ideal solution.

Thanks!

**Comment by jc-Rosichini** [comment]
[dnn]
library_path = C:\CUDA\v8.0\lib\**x64**

**Comment by jtc42** [comment]
Well, It works perfectly now. I can't believe I missed that. Thank you so much!

**Comment by cehkongfu** [comment]
dear jtc42:
     could you tell me how did you dissove the question about "In file included from C:\\CUDA\\v8.0\\include/driver_types.h:53:0,      
from C:\\CUDA\\v8.0\\include/cudnn.h:63,
from C:\\Users\\jtc92\\AppData\\Local\\Temp\\try_flags_733_4ik6.c:4:
C:\\CUDA\\v8.0\\include/host_defines.h:84:0: warning: "__cdecl" redefined
#define __cdecl <built-in>: 
note: this is the location of the previous definition"
